### ISSUE TITLE
Persist add-on settings

### ISSUE THREAD

**Issue by mehmetb** [mehmetb/dimmer#30]

**Comment by Andrew15-5** [comment]
Does this include the dimming level? Right now, my only struggle is that I don't like the overly dimmed default level, so I have to always click on the icon and move the slider to a less dimmed level. And if I have a lot of tabs opened, then I have to repeat this process for every tab. But for new tabs (maybe only in the same window) the less dimmed level persists, so at least I have that.

BTW, why the extension's window is white? Isn't the whole idea of it to make bright things less bright? Why not either apply dimmed effect on the extension window as well, or just use dark theme colors?

**Comment by mehmetb** [comment]
> Does this include the dimming level?

Yes, the default dim level will be configurable from the options page. And that will also be persisted along with other options.

> BTW, why the extension's window is white? Isn't the whole idea of it to make bright things less bright? Why not either apply dimmed effect on the extension window as well, or just use dark theme colors?

You are right. The design and colors are taken from the Firefox Photon Design example and they only had light theme colors. I'm reverse-engineering Firefox's dark and light theme colors for the next release. So, if you are using a light theme you'll see the current pop-up. But if your browser/operating system uses a dark theme you'll see the dark theme pop-up window. Still, applying the dim effect for the pop-up is a very good idea. So I'm adding this to the 'To Do' list (the dim effect will be applied to light theme users only) 👍
